sos/report/plugins/snap.py

```diff
@@ -28,4 +29,15 @@ def setup(self):
         self.add_service_status("snapd")
         self.add_journal(units="snapd")
 
+    def postproc(self):
+        self.do_file_sub(
+            "/var/lib/snapd/state.json",
+            (r"\"(macaroon|store-macaroon|key-id|session-macaroon"
+                r"|macaroon-key)\":\"([A-Za-z0-9_=-]*)\""),
+            r'"\1":"***"')
+        self.do_file_sub(
+            "/var/lib/snapd/state.json",
+            r"\"(store-discharges)\":\[([\"A-Za-z0-9_\-=,]*)\]",
+            r'"\1":[***]')
```

Comment:
Why not just pull `store-discharges` into the same `do_file_sub()` above? If we're going to be obfuscating the value anyways, do the brackets really matter?

Reply:
Yeah, since the PR allows edits from maintainers. Basically the flow is to clone their fork, update this branch locally, and then force push it back to *their* fork, which will update this PR.

If you're unsure about it, I can tackle updating this this weekend.